### PR TITLE
Fix copyright footer year forever

### PR DESCRIPTION
**Pull request by glepretre**
VanillaJS, n'altère pas le rendu.

| **before** | **after** |
|:----------:|:---------:|
|  Selection_004 |       Selection_005|

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -347,7 +347,15 @@ <h4>Une marque Kozea group</h4>
       </div>
 
       <div class="footer-lower">
-        <p><a href="https://www.kozea.fr/">©Kozea</a> 2021 - Tous droits réservés</p>
+        <p>
+          <a href="https://www.kozea.fr/">©Kozea</a> 
+          <span id="copyright-year">
+            <script type="text/javascript">
```
Comment (glepretre) [inline templates/index.html:353]: Without JavaScript: :frowning_face: 

image

**Comment by glepretre** [comment]
https://github.com/Kozea/www-pellov/pull/23#event-8057754993 Rebasing onto master

**Comment by glepretre** [comment]
@gentooboontoo 

call-me-fullstack

**Review by gentooboontoo — APPROVED** [review]
:+1:

**Comment by gentooboontoo** [comment]
> @gentooboontoo
> 
> call-me-fullstack

https://amiafullstackdevyet.com

**Pull request merged**